forge-gui/res/cardsfolder/upcoming/kroxa_and_kunoros.txt

```diff
@@ -0,0 +1,18 @@
+Name:Kroxa and Kunoros
+ManaCost:3 R W B
+Types:Legendary Creature Elder Giant Dog
+PT:6/6
+K:Vigilance
+K:Menace
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigImmediateTrig | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, you may exile five cards from your graveyard. When you do, return target creature card from your graveyard to the battlefield.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigImmediateTrig | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, you may exile five cards from your graveyard. When you do, return target creature card from your graveyard to the battlefield.
+SVar:TrigImmediateTrig:AB$ ImmediateTrigger | Cost$ ExileFromGrave<5/card> | Execute$ TrigReturn | SpellDescription$ When you do, return target creature card from your graveyard to the battlefield.
+SVar:TrigReturn:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Creature.YouOwn | TgtPrompt$ Select target creature to return from your graveyard
+DeckHas:Ability$Graveyard
+DeckHints:Ability$Discard
+SVar:NeedsToPlayVar:Z EQ6
```

Comment:
AI can play it just fine (because the trigger is optional?)

@Agetian can you check the AI Logic for the cost Trigger/Cost? it should not try to pay the cost if it doesn't have something "good" to reanimate?

Reply:
Is there an adjustment we can make for the AI logic in lieu of Agetian's input?

Reply:
When I made Young Necromancer I may have been holding the AI's hand too much

Reply:
Maybe for now just delete this line and we'll see if the AI can handle it or not